js/index.js

```diff
@@ -141,29 +141,27 @@ Raven.context(async () => {
     document.getElementById("app").style.visibility = "hidden";
     return;
   }
-  const __extraWindows = [];
-  let __initialWindowLayout = null;
 
+  let __butterchurnOptions = null;
+  let __initialWindowLayout = null;
   if (isButterchurnSupported()) {
     const startWithMilkdropHidden =
+      library ||
       document.body.clientWidth < MIN_MILKDROP_WIDTH ||
       (!library && skinUrl != null) ||
```

Comment:
I guess we don't need the `!library` here anymore.

Reply:
I’ll clean it up

Reply:
haha, OK sounds good, thanks!